pkg/controller/machineset/machineset_controller.go

```diff
@@ -99,6 +105,8 @@ func NewController(
 		logger:     logger,
 	}
 
+	c.machineLister = machineInformer.Lister()
```

Comment:
We need to add an event handler for machines. That will be used by the expectations to track when a machine is added or deleted.